java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/common/DbzConnectorConfig.java

```diff
@@ -319,11 +319,17 @@ public DbzConnectorConfig(
         } else {
             throw new RuntimeException("unsupported source type: " + source);
         }
-
         var otherProps = extractDebeziumProperties(userProps);
         for (var entry : otherProps.entrySet()) {
-            dbzProps.putIfAbsent(entry.getKey(), entry.getValue());
+            // For time.precision.mode, user configuration should override default configuration
```

Comment:
Can you include more contexts in the comment here so that people can understand why we treat time.precision.mode special? If we have a issue link, including it should be sufficient.